.github/workflows/release.yml

```diff
@@ -1,8 +1,8 @@
 name: Release
 
 on:
-  push:
-    tags: ['*.*.*']
+  release:
```

Comment:
this will only trigger when actually publishing releases - not just if a tag matches to be more explicit